namerd/iface/interpreter-thrift/src/main/scala/io/buoyant/namerd/iface/ThriftInterpreterInterfaceConfig.scala

```diff
@@ -51,7 +50,7 @@ class ThriftInterpreterInterfaceInitializer extends InterfaceInitializer {
   val configClass = classOf[ThriftInterpreterInterfaceConfig]
 }
 
-case class ThriftServable(addr: InetSocketAddress, iface: ThriftService, params: Stack.Params) extends Servable {
+case class ThriftServable(addr: InetSocketAddress, iface: AnyRef, params: Stack.Params) extends Servable {
```

Comment:
Could we still keep the strict type for `iface` as ThriftService using `import com.twitter.finagle.thrift.ThriftService`. Is there a reason why this should be `AnyRef`.

Reply:
com.twitter.finagle.thrift.ThriftService no longer exists